boa/src/builtins/object/mod.rs

```diff
@@ -109,6 +121,110 @@ impl Object {
         Ok(obj)
     }
 
+    /// `Object.getOwnPropertyDescriptor( object, property )`
+    ///
+    /// Returns an object describing the configuration of a specific property on a given object.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://www.ecma-international.org/ecma-262/10.0/index.html#sec-object.getownpropertydescriptor
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptor
+    pub fn get_own_property_descriptor(
+        _: &Value,
+        args: &[Value],
+        ctx: &mut Context,
+    ) -> Result<Value> {
+        let object = args.get(0).unwrap_or(&Value::undefined()).to_object(ctx)?;
+        if let Some(key) = args.get(1) {
+            let key = key.to_property_key(ctx)?;
+
+            if let Some(desc) = object.borrow().get_own_property(&key) {
+                return Ok(Self::from_property_descriptor(desc, ctx)?);
+            }
+        }
+
+        Ok(Value::undefined())
+    }
+
+    /// `Object.getOwnPropertyDescriptors( object )`
+    ///
+    /// Returns all own property descriptors of a given object.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://www.ecma-international.org/ecma-262/10.0/index.html#sec-object.getownpropertydescriptor
```

Comment:
Could we link the rolling spec instead of one specific version? Also, this one seems to go to the `getOwnPropertyDescriptor()` function.
https://tc39.es/ecma262/#sec-object.getownpropertydescriptors